fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -163,10 +189,12 @@ ACTOR Future<Standalone<RangeResultRef>> SpecialKeySpace::getRangeAggregationAct
 	if (reverse) {
 		while (iter != ranges.begin()) {
 			--iter;
+			onModuleRead(ryw, iter->value(), lastModuleRead);
 			if (iter->value() == nullptr) continue;
 			KeyRangeRef kr = iter->range();
 			KeyRef keyStart = kr.contains(begin.getKey()) ? begin.getKey() : kr.begin;
 			KeyRef keyEnd = kr.contains(end.getKey()) ? end.getKey() : kr.end;
+			onModuleRead(ryw, iter->value(), lastModuleRead);
```

Comment:
Why do we need to check `onModuleRead()` again here since L192 has done that?

Reply:
Since we actually found a module this time we need to update lastModuleRead, so that the check that we actually found a module (lastModuleRead != nullptr) can pass

Reply:
I think you're right, it's not necessary. Removed.